### ISSUE TITLE
Not supported in new version of Middleman and undocumented

### ISSUE THREAD

**Issue by marcinant** [middleman/middleman-sprockets#99]
I did what's mentioned in README  and my Middleman crashed with:

```
Tried to activate old-style extension: sprockets. They are no longer supported. (RuntimeError)
```

Could you please update README to avoid such confusion?

**Comment by stevenosloan** [comment]
Yeah I'll definitely add a note about `master` being a pre-released version of the extension.

Out of curiosity though, what versions of middleman/middleman-sprockets are you running? I wouldn't have expected that specific error except on any version since middleman-sprockets 3.2

**Comment by tdreyno** [comment]
Maybe it's a combo of `activate` and auto activation? Did we remove the auto from `master`?

**Comment by stevenosloan** [comment]
Yeah, auto is removed from master. I thought that was the error for any pre Middleman 3.1 extensions though from Middleman 3.2+?

**Comment by tdreyno** [comment]
Yeah, it is... but maybe there's a bug where double-active takes it down that code path? I'll see if I can re-produce.

**Comment by lawso017** [comment]
@stevenosloan I just posted the following over in the forms regarding a quick and easy setup to get middleman-sprockets running with rails-assets to manage frontend dependencies through Bundler:

https://forum.middlemanapp.com/t/asset-pipelining/2101/9?u=lawso017

Thought it might be helpful to add something along those lines to `middleman` or `middleman-sprocket`'s base documentation.

Long story short, I think rails-assets.org's recent sponsorship and growth make sprockets + middleman-assets a simple and stable solution for managing frontend assets without introducing entirely new toolsets.

**Comment by stevenosloan** [comment]
@lawso017 ah, good to know -- last I had looked into rails-assets was when EOL was being discussed. Glad it's sticking around. PR for docs would be great, probably best at `docs/usage_with_rails-assets.md` & linked to from the [readme](https://github.com/middleman/middleman-sprockets#usage).

(side note, closing as the opening issue has been resolved with  [19c07c1ff277](https://github.com/middleman/middleman-sprockets/commit/19c07c1ff277e8c0a32c7fecae5e41711b618316))
